fractions/javaee/undertow/module.conf

```diff
@@ -3,3 +3,4 @@ org.wildfly.extension.undertow
 io.undertow.core
 io.undertow.servlet
 org.ow2.asm
+org.wildfly.swarm.security
```

Comment:
Example of this cross-fraction optionality:

https://github.com/wildfly-swarm/wildfly-swarm/blob/master/fractions/wildfly/infinispan/src/main/java/org/wildfly/swarm/infinispan/runtime/InfinispanCustomizer.java#L45-L55

Notice those injections are bare config-api classes, so require no mvn (or module) dependency across them.